packages/workbox-build/src/lib/utils/runtime-caching-converter.js

```diff
@@ -0,0 +1,73 @@
+const errors = require('../errors');
+
+/**
+ * Given a set of options that configures `sw-toolbox`'s behavior, convert it
+ * into a string that would configure equivalent `workbox-sw` behavior.
+ *
+ * @param {Object} options See
+ *        https://googlechrome.github.io/sw-toolbox/api.html#options
+ * @return {String} A JSON string representing the equivalent options.
+ */
+function getOptionsString(options) {
+  const cacheOptions = options.cache || {};
+  // Start with a base of a few properties that need to be renamed, as well
+  // as copying over all the other source properties as-is.
+  const effectiveOptions = Object.assign({
+    cacheName: cacheOptions.name,
+  }, options);
+
+  // Only create the cacheExpiration object if either maxEntries or
+  // maxAgeSeconds is set.
+  if (cacheOptions.maxEntries || cacheOptions.maxAgeSeconds) {
+    effectiveOptions.cacheExpiration =
+      Object.assign(effectiveOptions.cacheExpiration || {}, {
+        maxEntries: cacheOptions.maxEntries,
+        maxAgeSeconds: cacheOptions.maxAgeSeconds,
+      });
+  }
+
+  // Everything should be copied to the corresponding new option names at this
+  // point, so delete the old-style `cache` property if it's present.
+  delete effectiveOptions.cache;
```

Comment:
If this was `effectiveOptions.cache = null` or `undefined`, would it be just as effective for Line 35? `delete` used like this will likely cause V8 deopts.